### PR TITLE
goerli-nightly-deployment

### PR DESCRIPTION
**Pull request by optimisticben**
**Description**
Contract deployment artifacts for a goerli-nightly stack.

### CODE REVIEW

File: packages/contracts/deployments/goerli-nightly/.chainId [hunk 1]
```diff
@@ -0,0 +1 @@
+5
```
Comment (optimisticben) [inline packages/contracts/deployments/goerli-nightly/.chainId:1]: hmm, should this have been changed?

Comment (smartcontracts) [reply packages/contracts/deployments/goerli-nightly/.chainId:1]: yep that's fine

File: packages/contracts/deploy-config/goerli-nightly.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const config: DeployConfig = {
+  network: 'goerli-nightly',
+  l1BlockTimeSeconds: 15,
+  l2BlockGasLimit: 15_000_000,
+  l2ChainId: 421,
+  ctcL2GasDiscountDivisor: 32,
+  ctcEnqueueGasCost: 60_000,
+  sccFaultProofWindowSeconds: 604800,
+  sccSequencerPublishWindowSeconds: 12592000,
+  ovmSequencerAddress: '0xba517B809d22D5e27F607c03dEBDe09d5Ad27049', // from 1pass testnet keys vault
```
Comment (smartcontracts) [inline packages/contracts/deploy-config/goerli-nightly.ts:12]: Can prob remove these comments

**Review by smartcontracts — COMMENTED** [review]
lgtm but worth removing those comments

File: packages/contracts/deploy-config/goerli-nightly.ts (hunk 2 above)
Comment (optimisticben) [reply packages/contracts/deploy-config/goerli-nightly.ts]: removed

**Review by smartcontracts — APPROVED** [review]

**Comment by tynes** [comment]
Looks good to me!

**Review by tynes — APPROVED** [review]

**Pull request merged**